docs/team/yeojunwei.md

```diff
@@ -8,4 +8,14 @@ title: Yeo Jun Wei's Project Portfolio Page
 TutorAid is a desktop application used for helping tech-savvy private tutors, who have busy schedules / many students, to keep track of each students' details.
 The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java.
 
-Given below are my contributions to the project.
\ No newline at end of file
+Given below are my contributions to the project:
```

Comment:
LGTM

Reply:
Agree with this, but maybe we can also standardise the way we are recording our contributions in this page across the team before merging?

Reply:
Yes agree, we can consider standardizing the format for recording of contributions.

Reply:
I have updated the project contributions, thanks for the suggestion!